data/src/client.rs

```diff
@@ -411,6 +411,15 @@ impl Client {
                 let nick = args.first()?;
                 self.resolved_nick = Some(nick.to_string());
 
+                // Loop on connect commands.
+                for command in self.config.on_connect.iter() {
+                    if let Ok(cmd) = crate::command::parse(command, None) {
+                        if let Ok(command) = proto::Command::try_from(cmd) {
+                            let _ = self.sender.try_send(command.into());
+                        };
+                    };
+                }
+
```

Comment:
Can we set this _after_ the nick password / ghost attempt? Just in-case user has that configured, so these commands are sent after they auth / recover